wqflask/wqflask/server_side.py

```diff
@@ -0,0 +1,74 @@
+# handles server side table processing
+
+
+
+class ServerSide(object):
```

Comment:
*nitpick:* Could you pick a better name for this class? The problem with this is that someone would have to maintain this in future, and it would be helpful to infer what this means from the name.

Reply:
I'm adding docstring to this class to explain it's purpose and usage. I guess that should do it, plus it seems redundant to name it something like `ServerSideProcessing`. WDYT?

Reply:
Maybe just ServerSideTable?

Reply:
@zsloan's is great imo! Also @Uditgulati the docstring idea is also a really good idea.